contracts/dtoken/src/lib.rs

```diff
@@ -2,13 +2,44 @@ use bigdecimal::{BigDecimal, ToPrimitive};
 use near_contract_standards::fungible_token::FungibleToken;
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::collections::UnorderedMap;
-use near_sdk::{near_bindgen, Balance};
+use near_sdk::json_types::U128;
+use near_sdk::require;
+use near_sdk::{
+    env, ext_contract, log, near_bindgen, serializer, AccountId, Balance, Gas, PanicOnDefault,
+    PromiseResult,
+};
 use std::str::FromStr;
 
-//near-sdk-3.1.0 AccountId => String used in FungibleToken
-//near-sdk-4.0.0-pre.4 AccountId => struct used in project
-//it line for version conflict fix
-pub type AccountId = String;
+const NO_DEPOSIT: Balance = 0;
+// Need to atach --gas=200000000000000 to 'borrow' call (80TGas here and 200TGas for call)
+const CONTROLLER_ACCOUNT_ID: &str = "ctrl.nearlend.testnet";
+const WETH_TOKEN_ACCOUNT_ID: &str = "dev-1639659058556-60126760016852";
+
+#[ext_contract(weth_token)]
+trait WethTokenInterface {
```

Comment:
@convowork1 I like idea to call it "Interface" let's stick to this style

WethTokenInterface -> Erc20Interface
ControllerFunctions  -> ControllerInterface